src/coreclr/jit/emitxarch.cpp

```diff
@@ -1213,6 +1213,7 @@ bool emitter::TakesEvexPrefix(const instrDesc* id) const
 #define DEFAULT_BYTE_EVEX_PREFIX_MASK 0xFFFFFFFF00000000ULL
 #define LBIT_IN_BYTE_EVEX_PREFIX 0x0000002000000000ULL
 #define LPRIMEBIT_IN_BYTE_EVEX_PREFIX 0x0000004000000000ULL
+#define EMBEDDED_BROADCAST_BIT 0x0000001000000000ULL
```

Comment:
This should probably be called `EVEX_B_BIT` or something, since we need to use it for 3 different scenarios

Reply:
Same general comment applies to most of the places we're using the term `EmbeddedBroadcast`. We probably want to differentiate between the general `EVEX.b` bit and the concept of `broadcast` in particular which requires looking at the instruction flags.